packages/mask/src/extension/popups/pages/Friends/Detail/UI.tsx

```diff
@@ -50,6 +52,25 @@ const useStyles = makeStyles()((theme) => ({
         fontSize: 12,
         color: theme.palette.maskColor.second,
     },
+    empty: {
+        position: 'absolute',
+        top: '50%',
+        left: '50%',
+        transform: 'translate(-50%,-50%)',
+        display: 'flex',
+        alignItems: 'center',
+        flexDirection: 'column',
+        gap: 12,
+        color: theme.palette.text.secondary,
```

Comment:
```suggestion
        color: theme.palette.maskColor.second,
```

Please confirm if the color is correct. The colors we use are all in the `maskColor` in the figma.